Sources/AppStoreConnectCLI/Commands/Users/Invitations/ListUserInvitationsCommand.swift

```diff
@@ -43,7 +43,7 @@ struct ListUserInvitationsCommand: CommonParsableCommand {
         let api = try makeClient()
 
         let endpoint = APIEndpoint.invitedUsers(
-            limit: limit.map { [ListInvitedUsers.Limit.visibleApps($0)] } ?? [],
+            limit: limitVisibleApps.map { [ListInvitedUsers.Limit.visibleApps($0)] } ?? [],
```

Comment:
It seems `limit` takes an `Optional` array where `nil` is sent when we don't specify any limits
Therefore we can simplify this to:  

```suggestion
            limit: limitVisibleApps.map { [ListInvitedUsers.Limit.visibleApps($0)] },
```